backend/package.json

```
@@ -20,17 +20,18 @@
   "publishConfig": {
     "registry": "https://npm.pkg.github.com/"
   },
-  "main": "server.js",
+  "main": "server.mjs",
   "files": [
     "lib",
-    "server.js"
+    "server.mjs"
   ],
   "scripts": {
-    "serve": "cross-env NODE_ENV=development node server.js",
-    "lint": "eslint server.js . --max-warnings=0",
+    "serve": "cross-env NODE_ENV=development node server.mjs",
+    "lint": "eslint server.mjs . --max-warnings=0",
     "lint-sarif": "yarn lint --format=@microsoft/eslint-formatter-sarif",
-    "test": "cross-env NODE_ENV=test jest",
-    "test-coverage": "yarn test --coverage"
+    "test": "cross-env NODE_ENV=test node --experimental-vm-modules $(yarn bin jest)",
+    "test-coverage": "yarn test --coverage",
+    "generate-diagrams": "node diagramGenerator.js"
```

```suggestion
    "generate-dependency-diagrams": "node generate-dependency-diagrams.js"
```
Longer, but the extra clarity can be worth it

---

adressed in: b89fb45cf1d636e90f43a2151fdc44a0743f96d6